src/automations/checkInactivity.js

```diff
@@ -102,64 +107,72 @@ async function scrapeInactiveIssues(client, references, issues) {
   const iterator = issues[Symbol.iterator]();
 
   for (let issue of iterator) {
-    const inactiveLabel = issue.labels.find(label => {
```

Comment:
Why did you remove this? We still need to reject checking any issues with inactivity labels, regardless of their review status. Remove your deletions please.

Reply:
This is not deleted, it is just shifted down to a different line due to some declarations being shifted before the `needsReview` check because they needed to be declared to call `references.get(issueTag).review`